src/express-tooling.ts

```diff
@@ -82,18 +87,22 @@ function requireScopesMiddleware(scopes: string[],
  * @param options
  * @returns express middleware
  */
-function handleOAuthRequestMiddleware(options: MiddlewareOptions): RequestHandler {
+function handleOAuthRequestMiddleware(options: MiddlewareOptions,
+  logger?: Logger): RequestHandler {
 
   const {
     tokenInfoEndpoint,
     publicEndpoints
   } = options;
 
   if (!tokenInfoEndpoint) {
+    if (logger) {
+      logger.error('tokenInfoEndpoint must be defined');
+    }
     throw TypeError('tokenInfoEndpoint must be defined');
   }
 
-  return function(req: ExtendedRequest, res: Response, next: NextFunction) {
+  return function (req: ExtendedRequest, res: Response, next: NextFunction) {
```

Comment:
we do not use whitespaces here, do we?

Reply:
or use an arrow function :)

Reply:
to be honest: the functions look so 2016 - we should refactor generally in a follow up ticket to be consistent in coding style ;)